docs/getting-started/net-map-req.asciidoc

```diff
@@ -13,19 +13,14 @@ configure `source.geo` and `destination.geo` ECS fields for your indices.
 
 [float]
 [[kibana-index-pattern]]
-=== Create {kib} index patterns
+=== Create {kib} data views
 
-To display map data, you must define {kib}
-{kibana-ref}/tutorial-define-index.html[index patterns] (*Stack Management* ->
-*Index Patterns*) that match the names or glob patterns used to define
-Elastic Security indices.
-
-NOTE: The Elastic Security indices are defined in the `securitysolution:defaultIndex` field
+To display map data, you must define a {kib}
+{kibana-ref}/data-views.html[data view] (*Stack Management* ->
+*Data Views*) that includes one or more of the indices specified in the `securitysolution:defaultIndex` field
 (*{kib}* -> *Stack Management* -> *Advanced Settings* -> *`securitysolution:defaultIndex`*).
 
-For example, if you define an Elastic Security `servers-europe-*` glob pattern,
-to display map data for the matching indices you must also define a {kib} index
-pattern that matches `servers-europe-*`, such as `servers-*`.
+For example, to display data that is stored using the index pattern `servers-europe-*` on the map, you must use a {kib} data view whose index pattern matches `servers-europe-*`, such as `servers-*`.
```

Comment:
```suggestion
For example, to display data that is stored in the index pattern `servers-europe-*` on the map, you must use a {kib} data view whose index pattern matches `servers-europe-*`, such as `servers-*`.
```

Reply:
I'm a bit confused by this sentence because I'm not sure "using" is the correct verb to use here. That said, if I'm just not understanding the message in this sentence and you meant to write it this way, feel free to ignore my recommendation.

Reply:
I figure the data is stored in an index, not an index pattern. The index pattern just refers to an index, right? So the index pattern is used to define which index the data will be stored in. I could be wrong!

Reply:
```suggestion
For example, to display data that is stored in indices matching the index pattern `servers-europe-*` on the map, you must use a {kib} data view whose index pattern matches `servers-europe-*`, such as `servers-*`.
```